lib/tasks/ont_data.rake

```diff
@@ -2,24 +2,43 @@
 
 require_relative '../traction_graphql'
 
-namespace :ont_runs do
-  task create: :environment do
-    puts '-> Creating ONT runs using GraphQL'
-
-    create_plates(count: 5)
-    create_libraries(count: 5)
-    create_runs(library_count: 5)
+namespace :ont_data do
+  task :create, [:num] => :environment do |_t, args|
+    puts '-> Creating ONT data using GraphQL'
+
+    count = args[:num]&.to_i || 5
+
+    # create count plates
+    barcodes = create_plates(count: count)
+    Plate.where(barcode: barcodes)
+
+    # create count plates with libraries
+    barcodes = create_plates(count: count)
+    plates = Plate.where(barcode: barcodes)
+    create_libraries(plates: plates)
```

Comment:
It seems like you don't need to look the plates up for this?  It only needs the plate barcodes which you already had.  I don't really mind that you're passing the whole plate in here, but then in `create_runs` further down you pass in the library names and not the libraries themselves, so there's some inconsistency.